configtx/capabilities_test.go

```diff
@@ -67,13 +67,13 @@ func TestOrdererCapabilities(t *testing.T) {
 
 	c := New(config)
 
-	ordererCapabilities, err := c.OrdererCapabilities()
+	ordererCapabilities, err := c.OriginalConfig().Orderer().Capabilities()
```

Comment:
I may have said this already in a different section but I think removing the `Config` suffix from Original and Updated would probably be a good move.